### ISSUE TITLE
amenity=bench with tourism=yes/attraction tag is not rendered

### ISSUE THREAD

**Issue by onovy** [gravitystorm/openstreetmap-carto#5010]
### Expected behavior
Render as amenity=bench even with tags tourism=yes or tourism=attraction

### Actual behavior
Node is not rendered at all

### Screenshots with links illustrating the problem
[map](https://www.openstreetmap.org/#map=19/50.108042/14.413601)
[node](https://www.openstreetmap.org/node/6341954211)
![image](https://github.com/user-attachments/assets/118d1b1f-1965-4672-bac7-a24e55ba8422)

Thanks.

**Comment by imagico** [comment]
Thanks for the report.

As is this issue is invalid, the feature in question is not displayed because it collides with the information board (https://www.openstreetmap.org/node/10297047800), not because of the tagging.  `amenity=bench` with `tourism=yes`/`tourism=attraction` renders perfectly fine - see https://www.openstreetmap.org/node/10297047800, https://www.openstreetmap.org/node/11676877509.  `amenity=bench` with `tourism=attraction` and `name=*` displays a name label for the attraction rather than a bench symbol (https://www.openstreetmap.org/node/5174053067) - which is as intended - the attraction has priority over the bench.  It is generally not advisable to conflate several real world things into the same OSM element because of the ambiguous semantics (it is unclear if the name is the name of the bench or of an abstract attraction the bench happens to be located at).  Us rendering name labels for `tourism=attraction` without further qualification has for a long time been a doubtful strategy (because it is not generally a primary tag and often abused for arbitrary label placement).  I opened #5011 for that.

In the concrete case example the more suitable tagging by the way would probably be `tourism=artwork` (697 uses in combination with `amenity=bench`) or `historic=memorial`+`memorial=bench` (348 uses in combination with `amenity=bench`).  We could think about rendering a specific symbol variation for such combinations.  And the proper name would seem to be something like _Lavička Jaroslava Foglara číslo X_ because _Lavička Jaroslava Foglara_ is a classification (there are several of those) rather than an individual identifier.

Closing as invalid.
